src/client/dfuse/dfuse_core.c

```diff
@@ -1092,6 +1095,12 @@ dfuse_fs_start(struct dfuse_projection_info *fs_handle, struct dfuse_cont *dfs)
 	if (!args.argv[4])
 		D_GOTO(err, rc = -DER_NOMEM);
 
+	if (fs_handle->dpi_info->di_multi_user) {
+		args.argv[5] = strdup("-oallow_other");
```

Comment:
STRNDUP_S ?   Or does fuse actually free these internally in fuse_opt_free_args ?